kumo-core/src/main/java/com/kennycason/kumo/nlp/FrequencyAnalyzer.java

```diff
@@ -81,11 +81,55 @@ public FrequencyAnalyzer() {
         return load(Collections.singletonList(doc.body().text()));
     }
 
+
     public List<WordFrequency> load(final List<String> texts) {
         final List<WordFrequency> wordFrequencies = new ArrayList<>();
 
+
         final Map<String, Integer> cloud = buildWordFrequencies(texts, wordTokenizer);
         cloud.forEach((key, value) -> wordFrequencies.add(new WordFrequency(key, value)));
+        return takeTopFrequencies(wordFrequencies);
+    }
+
+
+    /**
+     *load data with auto fill function, if word exist, multiple the words. Else use nothing instead
+     * the first input parameter can be  InputStream or list of string
+     * @param autoFill decide whether or not to fill the cloud automatically
+     * @param autoFillWord  is used to define use what word to fill the cloud when there is no enough word.
+     */
+    public List<WordFrequency> load(final InputStream inputStream, boolean autoFill, String autoFillWord) throws IOException {
+        return load(IOUtils.readLines(inputStream, characterEncoding), autoFill, autoFillWord);
+    }
+    public List<WordFrequency> load(final InputStream inputStream, boolean autoFill) throws IOException {
+        return load(IOUtils.readLines(inputStream, characterEncoding), autoFill, "nothing");
+    }
+    public List<WordFrequency> load(final List<String> texts, boolean autoFill){
+        return load(texts, autoFill, "nothing");
+    }
+
+    public List<WordFrequency> load(final List<String> texts, boolean autoFill, String autoFillWord) {
+        if(!autoFill){
+            return load(texts);
+        }
+
+        final List<WordFrequency> wordFrequencies = new ArrayList<>();
+
+        final Map<String, Integer> cloud = buildWordFrequencies(texts, wordTokenizer);
+        int totalLength = 0;
+        for (Map.Entry<String, Integer> entry : cloud.entrySet()) {
+            totalLength += entry.getKey().length();
+        }
+        if (totalLength == 0){
+            cloud.put(autoFillWord,1);
+            totalLength = 7;
+            for (int i = 0; i < Math.max(wordFrequenciesToReturn / totalLength,1); i++) {
```

Comment:
```java
final int timesToAdd = Math.max(wordFrequenciesToReturn / totalLength, 1);
for (int i = 0; i < timesToAdd; i++) { ... 
```
I recommend pulling this condition out of the for-loop to help performance a bit.